src/instructlab/cli/model/evaluate.py

```diff
@@ -69,6 +70,12 @@ def set_benchmark_specific_vars(
     cls=clickext.ConfigOption,
     required=True,  # default from config
 )
+@click.option(
+    "--base-model-id",
```

Comment:
default from cfg?

Reply:
Let's leave it blank to keep logic simple